Components/Typography/typography.css

```diff
@@ -0,0 +1,117 @@
+.typ-ex-grid {
+  display: grid;
+  grid-template-columns: max-content min-content;
+  grid-template-rows: repeat(auto, 50px);
+  align-items: center;
+  grid-gap: 30px;
```

Comment:
grid-gap is deprecated and renamed as gap, it is working but using gap will be better practice.

https://stackoverflow.com/questions/49970635/property-grid-gap-doesnt-exist-in-css#:~:text='grid%2Dgap'%20has%20been,by%20Visual%20Studio%20yet%20either.

https://github.com/postcss/autoprefixer/issues/1046